tests/suites/test_suite_psa_crypto.function

```diff
@@ -3152,6 +3152,862 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void aead_multipart_encrypt( int key_type_arg, data_t *key_data,
+                             int alg_arg,
+                             data_t *nonce,
+                             data_t *additional_data,
+                             int ad_part_len,
+                             data_t *input_data,
+                             int data_part_len,
+                             data_t *expected_result )
```

Comment:
Naming: we tend to use “result” for a return value, not for an output. (Although `expected_ret` is preferred for mbedtls return values, and `expected_status` for psa return values.)